media/js/base/mozilla-sync-button.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+(function() {
+    'use strict';
+
+    var SyncButton = {};
+
+    SyncButton.init = function() {
```

Comment:
Yeah, this is that hacky repetition I wasn't happy with. We're now repeating the same functionality three times and the only real difference is what they're called. 

For this short-lived experiment I can get away with using `mozilla-monitor-button.js` and avoid introducing the third duplicate. But longer term I wouldn't want to use something named after Monitor for anything other than Monitor, since that's just confusing. 

In the very near future we should refactor that original Monitor button into a more general "FxA Service" button.